includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -403,6 +403,17 @@ public function payment_scripts() {
 			return;
 		}
 
+		// On product/cart pages where express checkout isn't enabled,
+		// only enqueue Stripe JS — skip the full UPE bundle.
+		if (
+			( is_product() && ! WC_Stripe_Helper::should_load_scripts_for_ece_location( 'product' ) )
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Helper' in method 'payment_scripts'. (undefined)

(StaticAccess)